src/common/personalizedDigest.ts

```diff
@@ -231,11 +232,13 @@ export const getPersonalizedDigestEmailPayload = async ({
   previousSendDate: Date;
   feature: PersonalizedDigestFeatureConfig;
 }): Promise<SendEmailRequestWithTemplate | undefined> => {
-  const feedConfig = await feedToFilters(
-    con,
-    personalizedDigest.userId,
-    personalizedDigest.userId,
-  );
+  const feedConfig = await await queryReadReplica(con, ({ queryRunner }) => {
```

Comment:
Yeah here it should be fine, the feed one is different story I guess, this one can be async/outdated even.